src/restorepackages.bat

```diff
@@ -1,4 +1,4 @@
-@echo off
+﻿@echo off
```

Comment:
are these the same changes as in your other PR?

Reply:
This branch was created from the other one, since I couldn't get it to build without it. :-)

Reply:
I guess @QilongTang approved the other fix, so once it's merged, it will just override this. We should be good.

Reply:
I have just merged the other PR, maybe merging latest R2018 branch into this branch will make this PR cleaner?

Reply:
will do. thanks!